stage1/init/kvm/hypervisor/lkvm_driver.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build hv_lkvm
+
+package hypervisor
+
+import (
+	"fmt"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/coreos/rkt/stage1/init/kvm"
+)
+
+var (
+	driverConfiguration = KvmHypervisor{
```

Comment:
Does `driverConfiguration` need to be a global variable? It seems it can be declared in `StartCmd()`.